File: problemas/iniciante/1180/1180.py
def main():
    N = int(input())
    X = list(map(int, input().split()))

    menorValor = min(X)
    posicao = X.index(menorValor)

    print(f"Menor valor: {menorValor}")
    print(f"Posicao: {posicao}")

if __name__ == "__main__":
    main()

File: problemas/iniciante/1180/1180.js
process.stdin.setEncoding('utf-8');

let input = '';

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let X = input.trim().split('\n')[1].split(' ').map(Number);

    const menorValor = Math.min(...X);
    let posicao = X.indexOf(menorValor);

    console.log("Menor valor: " + menorValor);
    console.log("Posicao: " + posicao);
});
